packages/esm-patient-forms-app/src/forms/forms.component.tsx

```diff
@@ -26,11 +27,14 @@ interface FormsProps {
 
 const Forms: React.FC<FormsProps> = ({ patientUuid, patient, pageSize, pageUrl, urlLabel, isOffline }) => {
   const { t } = useTranslation();
+  const config = useConfig() as ConfigObject;
   const headerTitle = t('forms', 'Forms');
   const isTablet = useLayoutType() === 'tablet';
   const [formsCategory, setFormsCategory] = useState(FormsCategory.All);
   const { isValidating, data, error } = useForms(patientUuid, undefined, undefined, isOffline);
-  const formsToDisplay = data?.filter((formInfo) => isValidOfflineFormEncounter(formInfo.form));
```

Comment:
**To be reviewed** - Do we really never want to include non offline valid forms within this list?
All HTML forms are currently considered invalid for offline so, they'll never show up, thus making the new configurable Configurable list of HFE forms useless.

cc @ibacher

Reply:
Hey, no, that is not intended. I created [a PR to address this](https://github.com/openmrs/openmrs-esm-patient-chart/pull/545).

Reply:
Oh, makes sense. Thanks!

Reply:
@icrc-psousa Could you update this branch? I've merged the PR @manuelroemer mentioned and I think that's causing a conflict.